runtime/gc_structs/ConstantPoolObjectSlotIterator.cpp

```diff
@@ -78,7 +78,7 @@ GC_ConstantPoolObjectSlotIterator::nextSlot() {
 		}
 
 		/* Adjust the CP slot and description information */
-		_cpEntry = (j9object_t *) (((U_8 *) _cpEntry)
+		_cpEntry = (j9object_t *) (((uint8_t *) _cpEntry)
```

Comment:
I tend to not leave spece between cast operatior and argument, but not sure what coding standard say (and what's more common) @dmitripivkine 

(j9object_t *)(((uint8_t *)_cpEntry)

Reply:
yes, agreed

Reply:
also we can remove unnecessary space lines 61, 64, 67